tests/CMakeLists.txt

```diff
@@ -67,6 +67,28 @@ add_test_case(resource_name_tostring_failure_test)
 add_test_case(resource_name_length_test)
 
 add_test_case(parse_ruleset_from_string)
+add_test_case(test_endpoints_aws_region)
+add_test_case(test_endpoints_default_values)
+add_test_case(test_endpoints_eventbridge)
+add_test_case(test_endpoints_fns)
+# no associated test cases
+#add_test_case(test_endpoints_get_attr_type_inference) 
+add_test_case(test_endpoints_headers)
+add_test_case(test_endpoints_is_virtual_hostable_s3_bucket)
+#no associated test cases
+#add_test_case(test_endpoints_region_override)
+#add_test_case(test_endpoints_minimal_ruleset)
```

Comment:
why comment them out? What do you mean by `no associated test cases`?

Reply:
some of the sample rulesets do not have associated test files. updated the test helper to skip resolve part in that case and just validate that parsing suceeds.